itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/AbstractLaunchConfigurationTest.java

```diff
@@ -0,0 +1,101 @@
+package org.jboss.tools.quarkus.integration.tests.project;
```

Comment:
Please also put it in different package, like it will be done in QuarkusLaunchConfigurationTabGroup.java.